### PR TITLE
org.fedoraproject.MediaWriter

### PR DESCRIPTION
**Pull request by MartinBriza**

### CODE REVIEW

File: org.fedoraproject.MediaWriter.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+	"--system-talk-name=org.freedesktop.UDisks2"
+    ],
+    "modules": [
+        {
+            "name": "MediaWriter",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "http://github.com/MartinBriza/MediaWriter",
+                    "branch": "master"
```
Comment (nedrichards) [inline org.fedoraproject.MediaWriter.json]: this needs to point to a stable release tag, commit or archive.

File: org.fedoraproject.MediaWriter.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        "--socket=x11",
+	"--device=dri",
+        "--filesystem=host",
+	"--share=network",
+	"--share=ipc",
+	"--system-talk-name=org.freedesktop.UDisks2"
+    ],
+    "modules": [
+        {
+            "name": "MediaWriter",
```
Comment (nedrichards) [inline org.fedoraproject.MediaWriter.json]: flatpak-builder newer than 0.9.11 has a qmake buildssytem, you don't need to pass a configure script.

File: org.fedoraproject.MediaWriter.json [hunk 3]
```diff
@@ -0,0 +1,34 @@
+{
+    "id": "org.fedoraproject.MediaWriter",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.9",
+    "sdk": "org.kde.Sdk",
+    "command": "mediawriter",
+    "rename-appdata-file": "mediawriter.appdata.xml",
+    "rename-desktop-file": "mediawriter.desktop",
+    "finish-args": [
+        "--socket=x11",
```
Comment (nedrichards) [inline org.fedoraproject.MediaWriter.json]: consistent indenting please

File: org.fedoraproject.MediaWriter.json [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+	"--device=dri",
+        "--filesystem=host",
```
Comment (nedrichards) [inline org.fedoraproject.MediaWriter.json]: consistent indenting please

File: build.sh [hunk 5]
```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
```
Comment (nedrichards) [inline build.sh]: flathub doesn't need this file.

**Comment by nedrichards** [comment]
Test build: https://flathub.org/builds/#/builders/1/builds/964

**Comment by nedrichards** [comment]
Huh, the x86-64 build failed with a transient error. Another build: https://flathub.org/builds/#/builders/1/builds/972

**Comment by MartinBriza** [comment]
Thanks for the review! I fixed everything you pointed out.

**Comment by nedrichards** [comment]
New test build: https://flathub.org/builds/#/builders/1/builds/985

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/org.fedoraproject.MediaWriter

**Pull request closed without merging**